extensions/wikia/Recirculation/services/CakeRelatedContentService.php

```diff
@@ -163,7 +163,8 @@ private function getAuthor( Content $content ) {
 			return "";
 		}
 
-		return $authorObjects[0]->getUsername();
+		return array("username" => $authorObjects[0]->getUsername(),
+					"avatar" => $authorObjects[0]->getAvatar());
 	}
 
 	private function getRecirculationContentType( $contentType ) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Reduce the number of returns of this function 4, down to the maximum allowed 3. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1142)